docs/CONFIG-VARS.md

```diff
@@ -327,6 +327,16 @@ V4_CFG_POSTGRES_SERVERS:
 | V4_CFG_CONNECT_ENABLE_LOADBALANCER | Set up LoadBalancer to access SAS/CONNECT | bool | false | false | | viya |
 | V4_CFG_CONNECT_FQDN | FQDN that is assigned to access SAS/CONNECT | string | | false | Required when V4_CFG_TLS_MODE is not disabled and cert-manager is used to issue TLS certificates. This FQDN is added to the SAN DNS list of the issued certificates. | viya |
 
+## Workload Orchestrator
+
+| Name | Description | Type | Default | Required | Notes | Tasks |
+| :--- |------------:| ---: | ---: | ---: | ---: | ---: |
+| V4_WORKLOAD_ORCHESTRATOR_ENABLED | Enables the [SAS Workload Orchestrator](https://go.documentation.sas.com/doc/en/itopscdc/default/dplyml0phy0dkr/n08u2yg8tdkb4jn18u8zsi6yfv3d.htm#p1vo217m7ffso5n11vxwsyycw4tg) service and configures the required ClusterRole and ClusterRoleBinding used by the daemon. Setting this to false will disable SAS Workload Orchestrator service entirely | bool | true | false | This flag is only applicable for cadences 2023.08 and newer, this flag will perform no action on older cadences. | viya |
```

Comment:
Will likely want to updated this SAS document URL to not use the go. prefix and also put it through the React URL converter [here](https://docportal.unx.sas.com/url-conversion/) to get this format instead
https://documentation.sas.com/?cdcId=itopscdc&cdcVersion=default&docsetId=dplyml0phy0dkr&docsetTarget=n08u2yg8tdkb4jn18u8zsi6yfv3d.htm#p1vo217m7ffso5n11vxwsyycw4tg

Reply:
I'm expecting that we will eventually want to update the links to use the parameterized format from the converter tool, like the I put in the comment above.  I'm going to resolve these comments since we haven't come to a consensus yet.

Reply:
nm, I didn't see the last commit, looks like I'd expect now.